### ISSUE TITLE
"Install" Admin. Possible?

### ISSUE THREAD

**Issue by cfoellmann** [johnjamesjacoby/wp-multi-network#42]
Does anyone see a way to create an admin area one level above `wp-admin/network`? And this in a "(semi-)native" way?

**Comment by johnjamesjacoby** [comment]
This would be ideal, and in previous discussions we've referred to this as a "Global" dashboard.

See also: https://core.trac.wordpress.org/ticket/32754

**Comment by johnjamesjacoby** [comment]
I have a pretty radical idea for this I'd like to implement in 1.8.0.

**Comment by JJJ** [comment]
Paging @felixarntz who is also working on a "global" admin area.

**Comment by felixarntz** [comment]
@JJJ I'm still interested in working on this, building it into WP Multi Network is a great idea. I initially started this in https://github.com/felixarntz/global-admin, but development is on hold since there are some problems with Core support that can't be hacked around properly.

I opened https://core.trac.wordpress.org/ticket/37526 a while back, and that would be what needs to happen before we can really get such a project started. Unfortunately there hasn't really been much feedback yet, I assume since it's a very specific idea. It would be great if we can get this rolling for 4.8.
